paimon-core/src/test/java/org/apache/paimon/rest/RESTCatalogServer.java

```diff
@@ -606,6 +616,29 @@ private Optional<MockResponse> checkTablePartitioned(Identifier identifier) {
                         new ErrorResponse(ErrorResponse.RESOURCE_TYPE_TABLE, null, "", 404), 404));
     }
 
+    private MockResponse authTable(Identifier identifier, String data) throws Exception {
+        AuthTableQueryRequest requestBody =
+                OBJECT_MAPPER.readValue(data, AuthTableQueryRequest.class);
+        if (noPermissionTables.contains(identifier.getFullName())) {
+            throw new Catalog.TableNoPermissionException(identifier);
+        }
+        if (!tableMetadataStore.containsKey(identifier.getFullName())) {
+            throw new Catalog.TableNotExistException(identifier);
+        }
+        List<String> columnAuth = columnAuthHandler.get(identifier.getFullName());
+        if (columnAuth != null) {
+            requestBody
+                    .select()
+                    .forEach(
+                            column -> {
+                                if (!columnAuth.contains(column)) {
+                                    throw new Catalog.TableNoPermissionException(identifier);
+                                }
+                            });
```

Comment:
I'm not sure requestBody.filter() how to check.

Reply:
Deserialize filters to `List<Predicate>` and do check.

For example, f1>0, If we configure row level permissions on the server, such as only supporting user A to obtain data with f1>0, then the filter here must contain f1>0 or f1>5 in order to pass the verification.

Reply:
Looks reasonable.